build/gitlab/stages/3-cluster-connect/cluster-connect

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+time="$(TZ=IST date)"
+current_time=$time
+echo $current_time
+path=$(pwd)
+
+printf "***Conneting to the Cluster*****\n\n"
+
+##Conneting to the Cluster
+go="sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'ls | grep go'"
+if [ -z "$go" ] 
+then
+  echo "GO is not setup. please setup GO first"
+  exit 1;
+fi
+echo "Printing home directory"
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'ls'
```

Comment:
Do we need this

Reply:
displaying info of home dir
can remove also.

Reply:
Removed